services/metalv1/docs/SelfServiceReservationItemRequest.md

```diff
@@ -4,11 +4,10 @@
 
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**Amount** | Pointer to **float32** |  | [optional] 
```

Comment:
This is removing declaration of the `amount` property on a self-service reservation request; support for that attribute has already been removed from the API itself.